hawkbit-ui/src/main/java/org/eclipse/hawkbit/ui/rollout/rollout/RolloutListGrid.java

```diff
@@ -132,6 +132,8 @@
                 uiNotification, uiProperties, entityFactory, i18n, eventBus, targetFilterQueryManagement);
         this.uiNotification = uiNotification;
         this.rolloutUIState = rolloutUIState;
+
+        init();
```

Comment:
Constructor should not call overridables see: http://www.javapractices.com/topic/TopicAction.do?Id=215